src/gui/widgets/Knob.cpp

```diff
@@ -172,6 +173,27 @@ void Knob::setLabel( const QString & txt )
 					horizontalAdvance(QFontMetrics(pointSizeF(font(), 6.5)), m_label)),
 						m_knobPixmap->height() + 10);
 	}
+
+	update();
+}
+
+
+void Knob::setHtmlLabel(const QString &htmltxt)
+{
+	m_label = htmltxt;
+	m_isHtmlLabel = true;
+	// Put the rendered HTML content into cache
+	m_tdRenderer = new QTextDocument();
+
+	// TODO: stop using <font>
+	m_tdRenderer->setHtml(QString("<font color=\"%1\">%2</font>").arg(textColor().name(), m_label));
```

Comment:
Maybe something like this?
```suggestion
	m_tdRenderer->setHtml(QString("<span style=\"color:\"%1\;">%2</span>").arg(textColor().name(), m_label));
```